packages/template-retail-react-app/app/pages/product-list/index.test.js

```diff
@@ -58,12 +57,6 @@ const MockedEmptyPage = () => {
 }
 
 beforeEach(() => {
-    useWishlist.mockReturnValue({
-        isInitialized: true,
-        isEmpty: false,
-        data: {},
-        findItemByProductId: () => {}
-    })
```

Comment:
Why this deletion?

Reply:
This is the mock for old hook, not the one we use. I am having another PR to clean up all the old wishlist